docs/en/sql-reference/sql-statements/Data Manipulation/UPDATE.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
This document also need to be added to the sidebar

Reply:
done